### ISSUE TITLE
How to use protobuf for face classification

### ISSUE THREAD

**Issue by MaartenBloemen** [davidsandberg/facenet#226]
Hi,

I'm trying to use the protobuf from my trained model to classify people, i've been looking at alot of examples but I can't seem to figure it out on how to use the pb model. Anyone have an example script or any form of documentation to help me along?
Thanks in advance.

**Comment by MaartenBloemen** [comment]
Loading the model into a graph was no problem, sorry for the misconception about that. My problem is that I can't figure out on how to feed an image to the graph so it makes a prediction on who the face belongs to.
I'm looking for something like the Google image classifier[example](https://github.com/tensorflow/models/blob/master/tutorials/image/imagenet/classify_image.py), on line 155(the Google model has a Tensor named "softmax") is where i'm getting stuck, I don't know which tensor to use with the facenet model.
